packages/runtime/patch-import-map.mjs

```diff
@@ -0,0 +1,13 @@
+import { readFile, writeFile } from "fs/promises"
+
+const importMapPath = new URL("src/templates/vendor/import_map.json", import.meta.url)
+const importMap = JSON.parse(await readFile(importMapPath, "utf8"))
+
+const newMap = {
+  scopes: {
+    "../": importMap.imports,
```

Comment:
We're moving the `imports` field generated by Deno into the `../` scope, so it only applies to our Next code and not to other, Netlify-managed code like the bootstrap layer.